Sources/Data/Services/ExperienceStore/ExperienceStoreService.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ExperienceStoreService.swift
+//  RoverData
+//
+//  Created by Sean Rucker on 2018-05-03.
+//  Copyright © 2018 Rover Labs Inc. All rights reserved.
+//
+
+import Foundation
+import os
+
+class ExperienceStoreService: ExperienceStore {
+    func get(byID id: String) -> Experience? {
+        do {
+            guard let path = experienceLocalURL(id: id) else {
```

Comment:
Small stylistic thing but this doesn't need to be inside the do/catch and reduces the indentation level if you pull it out.